### ISSUE TITLE
Eval question

### ISSUE THREAD

**Issue by Agade09** [Saelyos/MeanMax#2]
Do you want to describe this part of your evaluation function more?

"The distance between the destroyer and a close tanker with the max carrying water."

I am curious about this. Do you have a "magic number" for "close" and pick the tanker with the most water in that circle around your destroyer?

**Comment by Saelyos** [comment]
Yes, the magic number is 6000 for "close". For all tankers with `distance(destroyer, tanker) < 6000` I choose the one who has the most carrying water and subtract in my evaluation the distance between the destroyer and this tanker. If there are no tanker around I subtract 6000.

The fun part is that it was initially a bug in my code (I though that the maximum possible distance on the map was 6000) ! When I realized it, I first didn't understand how it could work. But then it seemed logical to me not to go for a "full" tanker at the opposite of the map as the destroyer is really slow and would never have the time to kill the tanker, so I decided to keep this feature !
However I haven't tested a lot if that makes a real difference.
